aixplain/modules/agent/tool/model_tool.py

```diff
@@ -76,4 +76,11 @@ def __init__(
             model = model.id
         self.supplier = supplier
         self.model = model
-        self.function = function
\ No newline at end of file
+        self.function = function
+
+    def validate_model_tool():
```

Comment:
Since this method is inside a class, you need the `self` attribute. Moreover, since it is in the ModelTool class, I would simplify the name for: `def validate(self)`